### PR TITLE
VGB-13: User Model is created

### PR DESCRIPTION
**Pull request by Jonathan-Bonnin**
VGB-13: User Model is created

### CODE REVIEW

File: GiTinder.Tests/Models/UserTests.cs [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+namespace GiTinder.Tests.Models
+{
+    public class UserTests
+    {
+        [Fact]
+        public void CanCreateUser()
+        {
+            var user = new User();
+            Assert.False(user == null);
```
Comment (adamgyulavari) [inline GiTinder.Tests/Models/UserTests.cs]: @Jonathan-Bonnin `Assert.True(true)`

Comment (adamgyulavari) [reply GiTinder.Tests/Models/UserTests.cs]: but we can remove this test

**Review by adamgyulavari — CHANGES_REQUESTED** [review]

File: GiTinder.Tests/Models/UserTests.cs (hunk 1 above)
Comment (Jonathan-Bonnin) [reply GiTinder.Tests/Models/UserTests.cs]: This test has been deleted :)

**Comment by adamgyulavari** [comment]
@Jonathan-Bonnin resolve conflicts

**Review by adamgyulavari — APPROVED** [review]

**Pull request merged**